package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "storybook-addon-vue-info",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
I'll bump this when releasing the package (includes other changes). Please do not change this.